default/stringtables/en.txt

```diff
@@ -4979,7 +4976,20 @@ SitRep types only appear in the Filters list once an event has occurred that wou
 
 Ignoring is useful for hiding a reoccurring event, such as reminders to build a [[buildingtype BLD_GAS_GIANT_GEN]] at one planet.
 
-Blocking is useful if you find some types of SitReps unhelpful, such as once you have seen all of the [[encyclopedia BEGINNER_HINTS]].'''
+Blocking is useful if you find some types of SitReps unhelpful, such as once you have seen all of the [[encyclopedia BEGINNER_HINTS]].
+'''
+
+RESEARCH_TECH_GUIDE_TITLE
+Researching Technologies
+
+RESEARCH_TECH_GUIDE_SHORT_DESC
+Research and unlock new items
+
+RESEARCH_TECH_GUIDE_TEXT
+'''Completing [[metertype METER_RESEARCH]] on a technology that unlocks an item will only give an empire the knowledge of how to construct that item. Actually constructing such an item may have other requirements, such as access to a certain resource.
+If an item has been unlocked and is not available to build, the additional requirements can be checked by displaying the hover text on the items listing in the Production window (with [[PRODUCTION_WND_AVAILABILITY_UNAVAILABLE]] selected).
+
+Generally, items unlocked by researched [[encyclopedia ENC_TECH]] do not require [[metertype METER_SUPPLY]] line connections to be available or active (e.g. a building or a planetary meter improvement). However, if a tech which improves a ship part is researched (e.g. a weapon), the already-existing ships must be in supply range, stationary, and not in combat, in order to receive the upgrade during the following turn.'''
```

Comment:
a bit ambiguous: does an upgrade only happen on the turn following researching a tech, or can it happen any time there is a supply connection to a ship?